beater/connbeat.go

```diff
@@ -21,6 +21,11 @@ type Connbeat struct {
 	done chan struct{}
 }
 
+type ContainerInfo struct {
+	id       string
+	localIPs mapset.Set
```

Comment:
Do ips automatically get deduplicated with this set? I am not sure what the type of the elements in the Set is.